decidim-core/app/cells/decidim/collapsible_list_cell.rb

```diff
@@ -19,7 +19,7 @@ module Decidim
   #      hidden_elements_count_i18n_key: "my.custom.key",
   #      size: :small
   #    )
-  class CollapsibleListCell < Decidim::ViewModel
+  class CollapsibleListCell < AuthorCell
```

Comment:
This cell is meant to list any kind of things. Making it inherit from `Decidim::AuthorCell` only lets us show authors, and you're coupling it with the `render :withdraw` part in the view side. 

Can we find another way to achieve the same result?